Cartfile

```diff
@@ -1,3 +1,5 @@
+github "Alamofire/Alamofire" ~> 4.9.0
```

Comment:
What is Alamofire?

Reply:
AlamoFire is a popular networking library for Swift: https://github.com/Alamofire/Alamofire